doc/releasenotes/16_0_0_summary.md

```diff
@@ -40,6 +40,9 @@ In [PR #11097](https://github.com/vitessio/vitess/pull/11097) we introduced nati
 Orchestrator integration in `vttablet` was deprecated in the previous release and is deleted in this release.
 Consider using `VTOrc` instead of `Orchestrator`.
 
+The [`mysqlctl` command-line client](https://vitess.io/docs/16.0/reference/programs/mysqlctl/) had some leftover (ignored) server flags after the [v15 pflag work](https://github.com/vitessio/enhancements/blob/main/veps/vep-4.md). Those unused flags have now been removed. If you are using any of the following flags with `mysqlctl` in your scripts or other tooling, they will need to be removed prior to upgrading to v16:
```

Comment:
Let's just had a sub-heading here to have a nice separation with the `Orchestrator Integration Deletion` sub-section. Otherwise looks good to me.

Reply:
For instance `#### mysqlctl CLI`

Reply:
Ah, right! Good catch. Will do.